src/pages/TaskCard/TaskCard.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useState } from "react";
+import { Image, CircularProgress } from "@nextui-org/react";
+import mascot from "../../assets/finalHomepage.png";
+import { useNavigate } from "react-router-dom";
+import tcard from "./TaskCard.json";
+import TCard from "./Card";
+
+export default function TaskCard() {
+  const [resource, setResource] = useState({});
+  const [value, setValue] = React.useState(50);
+  const navigateTo = useNavigate();
+
```

Comment:
Here, you can now use `useParams` hook to get the task that was selected.

i.e
```
import { useParams } from 'react-router-dom';

const { task } = useParams();
```